report/WorkerReport.ts

```diff
@@ -90,9 +90,7 @@ const request = async (msg: BlockRequestMessage) => {
         },
     };
 
-    try {
```

Comment:
This try must be kept, it should catch errors while developing blocks.

Reply:
Really not sure why I removed this haha